api/container/containerv2/clusters.go

```diff
@@ -30,6 +30,7 @@ type ClusterCreateRequest struct {
 	DefaultWorkerPoolEntitlement string           `json:"defaultWorkerPoolEntitlement"`
 	CosInstanceCRN               string           `json:"cosInstanceCRN"`
 	WorkerPools                  WorkerPoolConfig `json:"workerPool"`
+	SecurityGroupIDs	         []string		  `json:"securityGroupIDs,omitepmty"`
```

Comment:
there is a typo in the `omitempty`. can we have a unit test for an empty `SecurityGroupIDs` please?
a `go fmt ./...` would fix the indentation too